cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -1,7 +1,12 @@
+var threeDS2utils = require('./threeds2-js-utils.js');
+
 const configuration = {
     locale: $('#currentLocale').val(), // Defaults to en_US
     originKey: originKey,
-    loadingContext: loadingContext
+    environment: loadingContext,
```

Comment:
should be 'test' or 'live', according to 3.0.0 documentation, even if it works apparently with the url as well

Reply:
Good one, let me change that